scripts/manage-dist-tags.mjs

```
@@ -0,0 +1,228 @@
+async function main() {
+  const args = process.argv.slice(2)
+  const command = args[0] || 'validate'
+
+  try {
+    switch (command) {
+      case 'validate':
+        displayDistTagStrategy()
+        const isValid = await validateDistTags(true)
+        testVersionGraduation()
+
+        if (isValid) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)